src/components/Input/index.tsx

```diff
@@ -8,7 +8,7 @@ import unavaliable from './unavailable.svg'
 import { Label } from '../Label'
 import { mapTestIdToProps } from '../../utils'
 
-export type InputProps = {
```

Comment:
some of the existing props can be cleaned up since they now come from HTMLInput props